src/plugins/intel_cpu/src/nodes/mkldnn_matmul_node.cpp

```diff
@@ -22,9 +22,357 @@
 #include "mkldnn_extension_utils.h"
 #include <common/primitive_hashing_utils.hpp>
 
-using namespace mkldnn;
+#include "emitters/jit_emitter.hpp"
+#include "emitters/jit_eltwise_emitters.hpp"
+#include "emitters/jit_mkldnn_emitters.hpp"
+#include "emitters/jit_load_store_emitters.hpp"
+
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
+using namespace mkldnn;
+using namespace mkldnn::impl::utils;
+using namespace mkldnn::impl::cpu;
+using namespace mkldnn::impl::cpu::x64;
+using namespace Xbyak;
+
+#define GET_OFF(field) offsetof(jit_matmul_args, field)
+
+
+template <cpu_isa_t isa>
+struct jit_uni_matmul_kernel_f32 : public jit_uni_matmul_kernel, public jit_generator {
```

Comment:
mb it is worth to empathize this Jit kernel is intended to solve only small MatMul problems.  jit_uni_small_matmul_kernel_f32 or smt like that.